### ISSUE TITLE
Consul

### ISSUE THREAD

**Issue by eggwhat** [SaintAngeLs/courier_app#54]
I don't know if it is an issue on my end but after a service is registered with consul, they get critical failure check because of ping in appsetting.json. They are failing this check
`"pingEnabled": true
    "pingEndpoint": "ping",
    "pingInterval": 3,
    "removeAfterInterval": 3`
So to make things work i had to disable ping for all services on my current working branch "mailing". `"pingEnabled": false,`. After such change everything works fine.

**Comment by eggwhat** [comment]
I also realized that port for consul in appsettings.json is not matching the port that service runs on. It's a case for almost all services.

**Comment by SaintAngeLs** [comment]
> I also realized that port for consul in appsettings.json is not matching the port that service runs on. It's a case for almost all services.

You are right, I do use the other port for this micro infrastracture services, also for the dev reasons there is no need to be sure the consul is accepting all the request. We  will update the settings for all the infrastructure services later.

**Comment by SaintAngeLs** [comment]
But it is ot fixing the issue. This solution hiddes the information

**Comment by eggwhat** [comment]
Then it is worth to notice that currently on development branch Customers, Orders, Parcels services have ping set to false

**Comment by SaintAngeLs** [comment]
Maybe there is a point it worth, in case the consul is running as a container image and the ping is set to false, all the listed microservices are just not visible.

**Comment by SaintAngeLs** [comment]
Still there may not be the moment to worry about, as It is only one microservice, and it will be possible to make the requests directly to host it will have. The one large issue to worry about is #112
